src/main/java/com/suse/salt/netapi/calls/runner/Manage.java

```diff
@@ -27,6 +27,15 @@ private Manage() { }
                 new TypeToken<List<String>>(){});
     }
 
+    public static RunnerCall<List<String>> up(Optional<Integer> timeout,
+            Optional<Integer> gatherJobTimeout) {
```

Comment:
Using `Optional` in most module functions is more because of lazyness. Overloading is always preferred but gets quickly out of hand in case of many optional parameters. We don't promise backwards compatibility before 1.0 but if its easy to provide in this case we should go for it.